### PR TITLE
Sci names permitted params

### PR DESCRIPTION
**Pull request by tygriffin**
Fixes [666](https://github.com/Growstuff/growstuff/issues/666#issuecomment-71002122)

### CODE REVIEW

**Comment by pozorvlak** [comment]
Hmmm, you seem to have accidentally committed some unrelated changes to db/schema.rb. I *think* my merging it was harmless...

File: db/schema.rb [hunk 1]
```diff
@@ -116,7 +117,7 @@
     t.string   "area_unit"
   end
 
-  add_index "gardens", ["owner_id"], name: "index_gardens_on_owner_id", using: :btree
+  add_index "gardens", ["owner_id"], name: "index_gardens_on_user_id", using: :btree
```
Comment (Skud) [inline db/schema.rb]: Err, we don't use "user" anywhere. We say "member" and the actual field here is owner_id. What is going on here?

File: db/schema.rb [hunk 2]
```diff
@@ -175,11 +176,11 @@
     t.boolean  "send_planting_reminder",  default: true
   end
 
-  add_index "members", ["confirmation_token"], name: "index_members_on_confirmation_token", unique: true, using: :btree
-  add_index "members", ["email"], name: "index_members_on_email", unique: true, using: :btree
-  add_index "members", ["reset_password_token"], name: "index_members_on_reset_password_token", unique: true, using: :btree
-  add_index "members", ["slug"], name: "index_members_on_slug", unique: true, using: :btree
-  add_index "members", ["unlock_token"], name: "index_members_on_unlock_token", unique: true, using: :btree
+  add_index "members", ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true, using: :btree
```
Comment (Skud) [inline db/schema.rb]: Again with the "user" thing. We don't use that terminology. What is the intent of this change?

File: db/schema.rb [hunk 3]
```diff
@@ -20,24 +20,24 @@
     t.string   "name",              null: false
     t.boolean  "is_paid"
     t.boolean  "is_permanent_paid"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",        null: false
```
Comment (Skud) [inline db/schema.rb]: What caused all these schema changes?

Comment (tygriffin) [inline db/schema.rb]: No clue. Just did a quick search on "user" in the migrations and nothing comes up before 2013 when the users table was renamed to members.

**Pull request merged**